tests/Unit/FeedTest.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Tests\Feature;
+namespace Tests\Unit;
```

Comment:
What makes this a `Unit` test for you?

Reply:
I don't really know; just a preference/feeling thing.. (Also see comment below)

Reply:
kk, then please change this back to a Feature test. For me, everything that uses HTTP is a feature test, only when I test a class / method in isolation it is a unit test for me.